backend/dataall/modules/datasets/api/dataset/resolvers.py

```diff
@@ -9,22 +9,25 @@
 from dataall.base.db.exceptions import RequiredParameter, InvalidInput
 from dataall.modules.dataset_sharing.db.share_object_models import ShareObject
 from dataall.modules.datasets_base.db.dataset_models import Dataset
-from dataall.modules.datasets_base.services.datasets_base_enums import DatasetRole
+from dataall.modules.datasets_base.services.datasets_base_enums import DatasetRole, ConfidentialityClassification
 from dataall.modules.datasets.services.dataset_service import DatasetService
+from dataall.base.config import config
+
+custom_confidentiality_mapping = config.get_property('modules.datasets.features.custom_confidentiality_mapping', {})
 
 log = logging.getLogger(__name__)
 
 
 def create_dataset(context: Context, source, input=None):
-    RequestValidator.validate_creation_request(input)
+    RequestValidator.validate_creation_request(input, context)
```

Comment:
Adding context here, so that the unit tests - which use the standard confidentiality levels - work properly.

Reply:
Refactored to remove this in the next update